src/map/script.c

```diff
@@ -22970,6 +22995,70 @@ BUILDIN(navigateto)
 }
 
 /**
+ * Clan System: Add a player to a clan
+ */
+BUILDIN(clan_join)
+{
+	struct map_session_data *sd = NULL;
+	int clan_id = script_getnum(st, 2);
+
+	if (script_hasdata(st, 3))
```

Comment:
As I mentioned above, this should probably be a RID. (note: scripters can always use `charid2rid()` if they have a Char ID and want a RID)